lib/conjur/rest.rb

```diff
@@ -0,0 +1,138 @@
+#
+# Copyright (C) 2015 Conjur Inc
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of
+# this software and associated documentation files (the "Software"), to deal in
+# the Software without restriction, including without limitation the rights to
+# use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+# the Software, and to permit persons to whom the Software is furnished to do so,
+# subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+# FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+# COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+# IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+require 'rest-client'
+require 'conjur/build_from_response'
+require 'conjur/cast'
+require 'conjur/escape'
+require 'conjur/log_source'
+
+module Conjur
+# A REST resource with Conjur-related functionality.
+# Base for all REST references in the library.
+  class REST < RestClient::Resource
+    # Instance methods separated into a module to support compatibility code.
+    # Please roll back into REST class on 5.0.
+    module InstanceMethods
+      include Conjur::Cast
+      include Conjur::Escape
+      include Conjur::LogSource
+
+      def default_options
+        {
+          verify_ssl: true,
```

Comment:
It's a recent change in https://github.com/rest-client/rest-client/commit/d7ddc7e928b8e429e2ccd1f53817fd2afdf14bd3 -- it was exactly the problem that it wasn't :)

So I guess since that fix is released in 1.8.0, we can drop our `initialize` patch and bump the rest-client again.